brainscore/benchmarks/__init__.py

```diff
@@ -176,6 +176,21 @@ def _experimental_benchmark_pool():
     from .cadena2017 import ToliasCadena2017PLS, ToliasCadena2017Mask
     pool['tolias.Cadena2017-pls'] = LazyLoad(ToliasCadena2017PLS)
     pool['tolias.Cadena2017-mask'] = LazyLoad(ToliasCadena2017Mask)
+    # V1 properties benchmarks
+    # V1 orientation benchmarks
+    from .marques2020_ringach2002 import MarquesRingach2002V1CV, MarquesRingach2002V1BW, MarquesRingach2002V1OPR,\
+        MarquesRingach2002V1OS
+    pool['dicarlo.Marques2020_Ringach2002.V1-circ_var'] = LazyLoad(MarquesRingach2002V1CV)
+    pool['dicarlo.Marques2020_Ringach2002.V1-bandwidth'] = LazyLoad(MarquesRingach2002V1BW)
+    pool['dicarlo.Marques2020_Ringach2002.V1-orth_pref_ratio'] = LazyLoad(MarquesRingach2002V1OPR)
```

Comment:
I would keep the identifiers short since they are what appears in the website. I can update the name of the functions. Are you planning on writing 'representation_dissimilarity_matrix' instead of RDM? 'partia_least_squares' instead of PLS?

Reply:
ok agree if they become too long